build/helpers/config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Provides simple user configuration options.
+ */
+
+import loconfig from '../../loconfig.json' assert { type: 'json' };
+import { merge } from '../utils/index.js';
+
+try {
+    const usrconfig = await import('../../loconfig.local.json', {
+        assert: { type: "json" },
+    });
+    usrconfig = usrconfig.default;
+
+    merge(loconfig, usrconfig);
+} catch (err) {
+    // do nothing
+}
```

Comment:
For some reason, the `usrconfig` constant object is not being merged properly. I suggest changing the `usrconfig` constant object back into a `let` statement to make it mutable and able to be merged with the other object.

```suggestion
let usrconfig;

try {
    usrconfig = await import('../../loconfig.local.json', {
        assert: { type: 'json' }
    });
    usrconfig = usrconfig.default;

    merge(loconfig, usrconfig);
} catch (err) {
    // do nothing
}
```

Reply:
I've reverted this section to use `let` instead of `const`.